lang/cs/Org.Apache.REEF.IMRU/API/UpdateResult.cs

```diff
@@ -50,22 +50,22 @@ internal bool IsNotDone
             get { return !_done; }
         }
 
-        internal bool HasMapInput
+        public bool HasMapInput
```

Comment:
Why do we need these to be `public` now? If this is needed, we need documentation for these fields.